File: Day290/findTheWinnderOfAnArrayGame.java
class Solution {
    public int getWinner(int[] arr, int k) {
        int win = arr[0];
        int temp = 0; 
        for (int i=1; i<arr.length; i++) {
            if (win > arr[i]) {
                temp ++;
            } else {
                temp = 1;
                win = arr[i];
            }
            if (temp == k) {
                return win;
            }
        }
        return win;
    }
}

File: Day290/findTheWinnderOfAnArrayGame.go
func getWinner(arr []int, k int) int {
    win := arr[0]
    temp := 0 
    for i:=1; i<len(arr); i++ {
        if win > arr[i] {
            temp ++
        } else {
            temp = 1
            win = arr[i]
        }
        if temp == k {
            return win
        }
    }
    return win
}